Riot/Managers/Analytics/Analytics.m

```diff
@@ -31,7 +31,29 @@
 NSString *const kAnalyticsE2eDecryptionFailureAction = @"Decryption failure";
 
 
-@import PiwikTracker;
+@import MatomoTracker;
+
+@interface MatomoTracker (MatomoTrackerMigration)
++ (MatomoTracker *)shared;
+
++ (void)migrateFromFourPointFourSharedInstance;
+@end
+
+@implementation MatomoTracker (MatomoTrackerMigration)
++ (MatomoTracker *)shared
+{
+    NSDictionary *piwikConfig = [[NSUserDefaults standardUserDefaults] objectForKey:@"piwik"];
+    MatomoTracker *matomoTracker = [[MatomoTracker alloc] initWithSiteId:piwikConfig[@"siteId"] baseURL:[NSURL URLWithString:piwikConfig[@"url"]] userAgent:@"iOSMatomoTracker"];
```

Comment:
I forgot to call migrateFromFourPointFourSharedInstance after this line, which should also be an instance method.